app/pages/Auction/BidHistory.js

```diff
@@ -33,14 +34,17 @@ export default class BidHistory extends Component {
 
     bids.forEach(bid => {
       order.push(bid.from);
+
+      const {month, day, year} = createAMPMTimeStamp(bid.date);
       map[bid.from] = {
-        date: createAMPMTimeStamp(bid.date),
+        date: bid.date ? `${month}/${day}/${year}` : '(pending)',
```

Comment:
we'll probably have to revisit date/time formatting in I18n friendly way at some point. not important to focus on here, just observing